YIF.Core.Domain/ApiModels/RequestApiModels/SpecialtyToInstitutionOfEducationPostApiModel.cs

```diff
@@ -13,5 +16,10 @@ public class SpecialtyToInstitutionOfEducationPostApiModel
         /// </summary>
         /// <example>a47e79d8-7e68-43c5-a5c9-1c6dff1e1693</example>
```

Comment:
maybe you should change examples on another that works?

Reply:
we have local dbs that's why you have another id